proposals/0311-task-locals.md

```diff
@@ -372,10 +339,10 @@ await Lib.wasabiPreference.withValue(.withWasabi) {
 And finally, if we wanted to set the `withWasabi` reference for most of the tasks in some scope, except one or two of them, we can compose the scopes to achieve this, as expected:
 
 ```swift
-await Lib.wasabiPreference.withValue(.withWasabi) {
+await Lib.$wasabiPreference.withValue(.withWasabi) {
   async let firstMeal = cookDinner()
   async let secondMeal = cookDinner()
-  let noWasabiMeal = await Lib.wasabiPreference.withValue(.withoutWasabi) { 
+  spawn let noWasabiMeal = Lib.$wasabiPreference.withValue(.withoutWasabi) {
```

Comment:
(If the other examples aren't going to use `spawn let`.)

```suggestion
  async let noWasabiMeal = Lib.$wasabiPreference.withValue(.withoutWasabi) {
```

Reply:
The other way around and I'm changing that now :) Spawn let will go into review shortly and we're confident in the name